app/src/main/java/com/example/internetshop/data/cache/TokenPreference.kt

```diff
@@ -10,15 +10,15 @@ import javax.inject.Inject
 private const val PREFERENCE_NAME = "TokenPreference"
 
 class TokenPreference @Inject constructor(context: Context) {
-    private val preference: SharedPreferences = context.getSharedPreferences(PREFERENCE_NAME, Context.MODE_PRIVATE)
+    private val preference: SharedPreferences =
+        context.getSharedPreferences(PREFERENCE_NAME, Context.MODE_PRIVATE)
     private var tokenKey: String = "SessionToken"
```

Comment:
`tokenKey` should be val, so no one can modify it. Also, would be better to make this field as `const val` in companion object. Because it is a constant actually

Reply:
fixed